claims_test.go

```diff
@@ -382,7 +400,8 @@ func TestValidateBoundClaims(t *testing.T) {
 		},
 
 		{
-			name: "invalid received claim expected value",
+			name:            "invalid received claim expected value",
+			boundClaimsType: "string",
 			boundClaims: map[string]interface{}{
 				"email": "d",
 			},
```

Comment:
Include a test case with `boundClaimsType == ""` since that's what all existing roles will be read as.

Reply:
Extedned an existing test case by removing the `bound_claims_type` entry from the json representation of a role, saving the role in storage and reading it back (same testing paatern as for the role_type attribute).
I completely missed the "upgrade" path, i.e. reading from storage an existing role. Thanks for catching it.